low-level/json_msgs/messages/base_msg.py

```diff
@@ -39,6 +41,21 @@ def __init__(self):
     def getJson(self):
         raise NotImplementedError("Subclasses should implement this!")
 
+    def add_common_resources(self, jsonMsg, request_type):
+        try:
+            if jsonMsg.get("message").get(
+                request_type).get("info") is None:
+                return
+            conf_list = ["site_id", "node_id", "rack_id", "cluster_id"]
+            conf = fetch_config(conf_list)

```

Comment:
fetch_config will only work for few keys(which are used at many places). The whole point for introducing it was you want to save lines on config call. Now, when common config are added here, what is the use of fetch_config?

Using Conf.get() for these four keys will much simpler and readable

Reply:
Done the changes.